deployment/cloud-run/test_routing_debug.py

```
@@ -5,80 +5,129 @@
+        # Patch functions to avoid actual initialization
+        with patch(f'{__name__}.Api') as mock_api, \
+             patch(f'{__name__}.Namespace') as mock_ns:
+            self.mock_api = mock_api
+            self.mock_ns = mock_ns
+
+            # Create Flask app
+            self.app = Flask(__name__)
+
+            # Register root endpoint BEFORE Flask-RESTX initialization
+            @self.app.route('/')
+            def root():
```

Ruff (0.12.2)

Missing return type annotation for private function `root`

(ANN202)